tests/model.test.js

```diff
@@ -383,6 +383,69 @@ describe('Model methods', () => {
     comment.save()
   })
 
+  test('save() method makes a PATCH request when method is set using `config`', async () => {
+    let post
+
+    axiosMock.onAny().reply((config) => {
+      const _post = post
+      delete _post._config
+
+      expect(config.method).toEqual('patch')
+      expect(config.data).toEqual(JSON.stringify(_post))
+      expect(config.url).toEqual('http://localhost/posts/1')
+
+      return [200, {}]
+    })
+
+    post = new Post({ id: 1, title: 'Cool!' })
+    await post.config({ method: 'PATCH' }).save()
+  })
+
+  test('save() method makes a POST request even when `config` set to PATCH', async () => {
```

Comment:
Why it should make a "POST"?

Reply:
My idea was to avoid breaking stuff. In case you set "PATCH" in config, but the model doesn't have an ID, it still make a "POST" request.

Reply:
@robsontenorio What do you think?

Reply:
Ok, i got it. Maybe clarify the test name.

Reply:
Yeah, I agree!

Reply:
Done in 5ba661d